exercises/practice/armstrong-numbers/canonical-data.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
As mentioned on your other PR, the `canonical-data.json` file should probably not be part of this repo (unless you really want to).

Reply:
Thanks! Removing it